ZelBack/src/services/appsService.js

```diff
@@ -3598,19 +3648,32 @@ async function updateAppGlobalyApi(req, res) {
   });
 }
 
-async function installTemporaryLocalApplication(req, res, applicationName) {
+async function installTemporaryLocalApplication(req, res) {
   try {
+    let { appname } = req.params;
+    appname = appname || req.query.appname;
+
+    if (!appname) {
+      throw new Error('No Flux App specified');
+    }
+    const isNodeConfirmed = await isNodeStatusConfirmed();
+    if (!isNodeConfirmed) {
+      throw new Error('Flux Node is not Confirmed. Aborting install.');
+    }
     const authorized = await serviceHelper.verifyPrivilege('adminandfluxteam', req);
     if (authorized) {
       const allApps = await availableApps();
-      const appSpecifications = allApps.find((app) => app.name === applicationName);
+      const appSpecifications = allApps.find((app) => app.name === appname);
       if (!appSpecifications) {
         throw new Error('Application Specifications not found');
       }
 
+      const tier = await nodeTier();
+      if (appname === 'KadenaChainWebNode' && tier === 'basic') {
+        throw new Error('KadenaChainWebNode is not allowed to install on Cumulus Node.');
```

Comment:
should be prevented by default as of not enough available space (?)

Reply:
For now yes (forgot about disk space), but If we start using machine specs instead of pre defined specs for tier, this will solve the situation. :)